matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/algorithms/megolm/MXMegolmEncryption.kt

```diff
@@ -137,11 +137,10 @@ internal class MXMegolmEncryption(
         return MXOutboundSessionInfo(sessionId, SharedWithHelper(roomId, sessionId, cryptoStore))
     }
 
-    /**
-     * Ensure the outbound session
-     *
-     * @param devicesInRoom the devices list
-     */
+    override suspend fun ensureOutboundSession(usersInRoom: List<String>) {
+        getDevicesInRoom(usersInRoom).allowedDevices
```

Comment:
it's not ensuring outbound session, no? it's just getting allowed devices

Reply:
Done, (silly mistake :/ I wish I tested before creating PR)

Reply:
Done.